cypress/integration/chi-js/modal.spec.js

```diff
@@ -0,0 +1,182 @@
+describe('chi-modal', function() {
+  before(() => {
+    cy.visit('http://localhost:8000/tests/js/modal.html');
+  });
+
+  describe('check base modal', () => {
+    it(`modal should be opened when we click launch demo modal button`, () => {
+      cy.get('button#modal-trigger-1').click();
+      cy.get('#modal-1')
+        .should('have.class', '-active')
+        .find('.-close')
+        .click()
+        .wait(550);
+    });
+
+    it(`modal should be closed when we click close button`, () => {
+      cy.get('button#modal-trigger-1').click();
+      cy.get('#modal-1')
+        .find('.-close')
+        .click()
+        .wait(550)
+        .should('be.not.visible');
+    });
+
+    it(`modal should be closed when we click cancel button`, () => {
+      cy.get('button#modal-trigger-1').click();
+      cy.get('#modal-1')
+        .contains('Cancel')
+        .click()
+        .wait(550)
+        .should('be.not.visible');
```

Comment:
This assertion looks like this:
`assert
expected <button.chi-button.-icon.-close> not to be visible`
Please make sure that the element you check the visibility of is the Drawer itself, not the close button

Reply:
Fixed in the latest commit